front/src/modules/timesStops/styles/_timesStopsDatasheet.scss

```diff
@@ -1,31 +1,79 @@
 .time-stops-datasheet {
+  padding-bottom: 16px;
+  border-radius: 10px;
+  box-shadow:
+    0px 1px 0px rgba(255, 255, 255, 1) inset,
+    0px 4px 7px -3px rgba(255, 171, 88, 0.17),
+    0px 2px 4px rgba(0, 0, 0, 0.22);
+  background-color: var(--ambiantB15);
+  --dsg-cell-background-color: var(--white100);
+  --dsg-cell-disabled-background-color: var(--white100);
+  --dsg-header-text-color: var(--grey50);
+  --dsg-header-active-text-color: var(--black100);
+
   .activeRow {
```

Comment:
Do you think you can encapsulate the css concerning a row in the class dsg-row  ? Like this:
```
.dsg-row {
    &.oddRow {
      --dsg-cell-background-color: var(--ambiantB5);
      --dsg-cell-disabled-background-color: var(--ambiantB5);
    }

    &.activeRow {
      font-weight: 600;
      color: var(--black100);
      .dsg-cell-gutter,
      .secondary-code-column {
        font-weight: normal;
      }
    }
  }
```

Reply:
Sure thanks!